src/modules/app/StateType.js

```diff
@@ -6,7 +6,7 @@ import ExtraModel from '../endpoint/models/ExtraModel'
 import EventModel from '../endpoint/models/EventModel'
 import CityModel from '../endpoint/models/CityModel'
 import LanguageModel from '../endpoint/models/LanguageModel'
-import DisclaimerModel from '../endpoint/models/DisclaimerModel'
+import BasePageModel from '../endpoint/models/BasePageModel'
```

Comment:
I would just name it PageModel. No need for a Base imo. 
@Schedulaar what do you think?

Reply:
Would be okay for me.

Reply:
yup

Reply:
@maxammann done.